src/config/ConfigService.js

```diff
@@ -0,0 +1,64 @@
+import Service from '../abc/Service';
+import Axios from 'axios';
+import ConfigReducer from './ConfigReducer';
+import { CHANGE_CONFIG } from '../actions';
+
+/*
+	Example of use:
+
+	module.exports = {
+		app: {
+			configUrl: 'configuration/config/',
+		},
+	}
+
+...
+
+	Example of config.json file content:
+
+	{"kibanaUrl":"http://1.1.1.1:5601/app/kibana", "appSettings":{"onetwothree":123}}
+
+*/
+
+export default class ConfigService extends Service {
+
+	constructor(app, serviceName="ConfigService") {
+		super(app, serviceName);
+		app.ReduxService.addReducer("config", ConfigReducer);
+	}
+
+	initialize() {
+		// Check on undefined configuration
+		if (this.App.Config._config.configUrl !== undefined) {
+			this.App.addSplashScreenRequestor(this);
+			this.Axios = this.App.axiosCreate(this.App.Config._config.configUrl);
+			this.Axios.get().then(response => {
+					// TODO implement check on status codes
+					if (response.statusText === 'OK') {
```

Comment:
You have to check status code (200), here.
You will resolve the TODO (above) by that as well.